cloud/pkg/cloudstream/tunnelserver.go

```diff
@@ -169,3 +184,30 @@ func (s *TunnelServer) Start() {
 		return
 	}
 }
+
+func (s *TunnelServer) updateNodeKubeletEndpoint(nodeName string) {
+	if err := wait.PollImmediate(retrySleepTime, nodeStatusUpdateRetry, func() (bool, error) {
+		getNode, err := client.GetKubeClient().CoreV1().Nodes().Get(context.Background(), nodeName, metav1.GetOptions{})
+		if errors.IsNotFound(err) {
+			klog.Warningf("node %s not found", nodeName)
+			return false, nil
+		}
+		if err != nil {
+			klog.Errorf("Failed while getting a Node to retry updating node KubeletEndpoint Port. error: %v", nodeName, err)
+			return false, err
+		}
+
+		getNode.Status.DaemonEndpoints.KubeletEndpoint.Port = int32(s.tunnelPort)
+		_, err = client.GetKubeClient().CoreV1().Nodes().UpdateStatus(context.Background(), getNode, metav1.UpdateOptions{})
+		if err != nil {
+			klog.Errorf("update node KubeletEndpoint Port failed with error: %s, node: %v, tunnelPort: %v",
+				err, getNode.Name, s.tunnelPort)
+			return false, nil
+		}
+		return true, nil
+	}); err != nil {
+		klog.Errorf("Update KubeletEndpoint Port of Node '%v' error: %v. ", nodeName, err)
+		os.Exit(1)
+	}
+	klog.Infof("update node KubeletEndpoint Port success. node: %s, tunnelPort: %v", nodeName, s.tunnelPort)
```

Comment:
```suggestion
	klog.V(2).Infof("Update node KubeletEndpoint Port successfully, node: %s, tunnelPort: %s", nodeName, s.tunnelPort)
```

Reply:
V(4) maybe more fine, or large scale edge nodes will print lots of success logs.

Reply:
Done